PayPalMessaging/src/main/java/com/braintreepayments/api/PayPalMessagingPageType.kt

```diff
@@ -0,0 +1,56 @@
+package com.braintreepayments.api
+
+import com.paypal.messages.config.PayPalMessagePageType
+
+/**
+ * Message location within an application
+ * Note: **This module is in beta. It's public API may change or be removed in future releases.**
+ */
+enum class PayPalMessagingPageType {
```

Comment:
Can we use the "@RequiresOptIn" annotation for all public beta classes in this module in addition to the doc strings notes?

Reply:
Ah I see there is a separate ticket for this - so doesn't need to be in this PR 👍